sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -14,7 +14,9 @@
     <PackageTags>management;virtual machine;compute;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        This is a public release of the Azure Compute SDK. Included with this release is the vmHealth property added to the VMInstanceView API.
```

Comment:
The version on line 12 of this file needs to be updated.

Reply:
> The version on line 12 of this file needs to be updated.

The version is updated now